core/src/main/java/org/infinispan/remoting/rpc/RpcManagerImpl.java

```diff
@@ -252,16 +252,17 @@ public <T> CompletionStage<T> invokeCommand(Address target, ReplicableCommand co
       return invocation.handle((response, throwable) -> updateStatistics(startTimeNanos, response, throwable));
    }
 
-   private void checkTopologyId(ReplicableCommand command) {
+   private void checkTopologyId(CacheRpcCommand command) {
+      // TODO how to handle? SingleRpcCommand was never impacted by this as it does not implement TopologyAffectedCommand
```

Comment:
This still needs to be invoked together with `setClusterTraceSpanAttributes` for each request. 
With the `SingleRpcCommand`, the code checked the wrapped command.